README.md

```diff
@@ -55,6 +55,8 @@ You can also install from this repo with `cargo`:
 
     cargo install --git https://github.com/quodlibetor/git-fixup
 
+Or use Nixpkgs. It's [in there](https://search.nixos.org/packages?channel=unstable&query=git-fixup).
```

Comment:
Could you include the command people would run? Something like this:

```markdown
Or use [Nixpkgs](https://search.nixos.org/packages?channel=unstable&query=git-fixup):

	<the exact thing users would type>
```

Reply:
In Nix a package can be obtained in several different ways. I feel It's appropriate to not provide any examples in contexts such as this.